### DISCUSSION TITLE
【iOS】panic happen when create engine

### DISCUSSION THREAD

**Discussion by simonsland** [google/filament#4937] [Q&A]
<img width="1482" alt="截屏2021-12-09 上午12 19 31" src="https://user-images.githubusercontent.com/17590947/145244377-267e6f21-542c-4641-8503-198aa85bb0a1.png">

I am trying to integrate the filament into my project through pod, but after the project running, it always crashed at Engine::Create. and the information shows "reason: Texture format 30 not supported on this platform", 30 means "RGBA8" format which is the default format. And when I integrate the filament into an empty project, it works fine. Is there any reason why this happen?

**Comment by bejado** [comment]
Which version of Filament are you using?
